src/server/handle-api.js

```diff
@@ -1,4 +1,4 @@
-import axios from 'axios'
+import fetch from 'isomorphic-fetch'
```

Comment:
It's available in this commit if we want to do anything with them in the future, for now though I think we can get away with mocking progress via `setInterval`.